### PR TITLE
Build breaks

### PR DESCRIPTION
**Pull request by Scottj1s**
fixed authoring break from vtbl optimization, temporarily disabled authoring tests and api tests failing on build agent

### CODE REVIEW

**Review by j0shuams — APPROVED** [review]

**Comment by Scottj1s** [comment]
unfortunately, we took the vtbl optimization work based on a private build, just before authoring tests were enabled - temporarily disabling to unblock the build

File: cswinrt/code_writers.h [hunk 1]
```diff
@@ -5080,7 +5080,8 @@ public static unsafe void DisposeAbiArray(object box) => MarshalInspectable<obje
                             {
                                 w.write(
                                     ".As<IUnknownVftbl>(GuidGenerator.GetIID(typeof(%).GetHelperType()))",
-                                    bind<write_type_name>(get_type_semantics(get_default_interface(type)), typedef_name_type::Projected, false));
+                                    bind<write_type_name>(get_type_semantics(get_default_interface(type)), 
+                                        settings.component ? typedef_name_type::ABI : typedef_name_type::Projected, false));
```
Comment (manodasanW) [inline cswinrt/code_writers.h:5084]: I feel like you want typedef_name_type::ccw here which should handle component and non component.

File: UnitTest/ApiCompatTests.cs [hunk 2]
```diff
@@ -79,9 +79,7 @@ public void TestGeolocation()
         {
             Assert.True(InvokeGeolocation().Wait(1000));
         }
-#endif
 
-#if ENABLE_WORKSTATION_TESTS
```
Comment (BenJKuhn) [inline UnitTest/ApiCompatTests.cs:84]: The first #if is unmatched in this PR. Intentional?

Comment (Scottj1s) [reply UnitTest/ApiCompatTests.cs:84]: the #ifs are matched - the PR doesn't show it well

**Pull request merged**